### PR TITLE
Document how to add multiple domains to a hub

### PR DESCRIPTION
**Pull request by GeorgianaElena**
Closes https://github.com/2i2c-org/pilot-hubs/issues/214

### CODE REVIEW

File: docs/configure.md [hunk 1]
```diff
@@ -148,6 +148,69 @@ So we want to manage authentication by:
...
+
+      ```yaml
+      - name: foo
+        (...)
+        auth0:
+          connection: google-oauth2
+          domain: foo.pilot.2i2c.cloud
+        (...)
+      ```
+    * In Auth0's **Application Settings page**:
```
Comment (yuvipanda) [inline docs/configure.md:203]: Aaah, can you automate this [here](https://github.com/2i2c-org/pilot-hubs/blob/master/auth.py#L50) instead?

**Review by yuvipanda — CHANGES_REQUESTED** [review]
A point of automation that should make this easier :) I want to try remove manual steps as much as possible - not everyone will have access to auth0.

File: docs/configure.md (hunk 1 above)
Comment (GeorgianaElena) [reply docs/configure.md:203]: Sure! I just followed the instructions in the error page and missed that function :sweat_smile:

Comment (GeorgianaElena) [reply docs/configure.md]: I just opened https://github.com/2i2c-org/pilot-hubs/pull/225 to address this :)

**Comment by yuvipanda** [comment]
\o/ love it!

**Pull request merged**